src/components/partials/help-guide-lb-business.js

```diff
@@ -88,7 +88,7 @@ export default () => html`
     </section>
   </div>
 
-  <div class="side-padding vertical-padding">
+  <div id="foodWithExtraReq" class="side-padding vertical-padding">
```

Comment:
is id="foodWithExtraReq"
all on the same page?
if so this is against dom spec which is id's must be unique.
please use a class instead and make id's unique.

Reply:
Changed Id to class